.github/workflows/scripts/kinds/verify-kinds.go

```diff
@@ -182,47 +175,32 @@ func die(errs ...error) {
 
 // verifyKind verifies that stable kinds are not updated once published (new schemas
 // can be added but existing ones cannot be updated)
-func verifyKind(registry *kindRegistry, kind kindsys.Kind, name string, category string, latestRegistryDir string) error {
+func verifyKind(registry *kindRegistry, maturity string, name string, category string, latestRegistryDir string) error {
```

Comment:
I'm not sure what's being verified here: do we even need this function?

Reply:
I deleted the number of schemas verification and I only left the maturity. Looks like that isn't necessary so I'd say nope.